README.md

```diff
@@ -1,2 +1,30 @@
-# scraped_page
+# ScrapedPage
+
 Write declarative scrapers in Ruby
+
+## Usage
+
+Create a subclass of `ScrapedPage` for each page you wish to scrape.
+
+```ruby
+require 'scraped_page'
+
+class ExamplePage < ScrapedPage
+  field :title do
+    noko.at_css('h1').text
+  end
+  
+  field :more_information do
+    noko.at_css('a')[:href]
+  end
+end
+
+puts ExamplePage.new('http://example.com').to_h
+# => { :title => "Example Domain", :more_information => "http://www.iana.org/domains/reserved" }
+```
+
+The default strategy for retrieving pages is to make an http request to retrieve them. If you'd also like to archive a copy of the page you're scraping into a git branch, you can pass a `:strategy` option to the constructor:
+
+```ruby
+ExamplePage.new('http://example.com', strategy: ScrapedPage::LiveRequestArchiveStrategy)
```

Comment:
Hmm, now you mention it perhaps it would be better to fold this into a datastructure, so we'd have something more like this:

```ruby
ExamplePage.new('http://example.com', strategy: :live_request_archive)

# Somewhere else
strategies = {
  live_request_archive: ScrapedPage::Strategy::LiveRequestArchive,
}
```

And yes, giving them their own namespace seems like a very sensible idea.